test/unit/browser/index.js

```diff
@@ -20,9 +20,9 @@ const { applyReporter } = require('../reporter');
 // opts
 const defaultReporterName = process.platform === 'win32' ? 'list' : 'spec';
 const optimist = require('optimist')
-	// .describe('grep', 'only run tests matching <pattern>').alias('grep', 'g').alias('grep', 'f').string('grep')
 	.describe('build', 'run with build output (out-build)').boolean('build')
 	.describe('run', 'only run tests matching <relative_file_path>').string('run')
+	.describe('glob', 'only run tests matching <pattern>').string('glob')
```

Comment:
This should be `runGlob` to match our other unit test types

Reply:
> This should be `runGlob` to match our other unit test types, and be described as "only run test _files_ matching..."

emmmm, but in https://github.com/microsoft/vscode/blob/89db3e46dd79c88d2f4f25940ff2dc3bc9932c86/test/unit/README.md?plain=1#L24 and https://github.com/microsoft/vscode/blob/89db3e46dd79c88d2f4f25940ff2dc3bc9932c86/test/unit/browser/index.js#L84, It seems that the supported parameter is `--glob`.   Has it been abandoned?

Reply:
It looks like in the electron test we allow both `glob` and `runGlob`, so I guess that's fine. I would add a similar `.alias('glob', 'runGlob')` just for consistency's sake